extension-client/src/command/workspace/estimateCost.ts

```diff
@@ -54,6 +55,16 @@ async function estimateCost(writeEmitter:any,closeEmitter:any): Promise<any> {
  var terminal = new Terminal(writeEmitter, closeEmitter);
  const API_KEY = 'IC_API_KEY=';
  try{
+    terminal.printHeading("Verifying tfcost binary");
+     try{
+         await tfcost.verifyBinary();
+         terminal.printSuccess( "Binary verified" );
+
+     }
+     catch{
+        terminal.printFailure("Please download tfcost binary & include it in your path." +'\r\n'+ "Link: https://github.com/IBM-Cloud/terraform-cost-estimator/releases");
```

Comment:
Never use `please` in programming display text unless it's really necessary. 
Let's change the message to , 

```
Install tfcost and try again. See https://github.com/IBM-Cloud/terraform-cost-estimator#using-the-cli for details
```

Lets also add a modal that shows message `This feature requires you to install tfcost. Details of installtion can be found at https://github.com/IBM-Cloud/terraform-cost-estimator#using-the-cli. Do you want to open the URL ?`  [using a modal ](https://code.visualstudio.com/api/references/vscode-api#window.showInformationMessage) 
Reference can be found in code [here](https://github.com/IBM-Cloud/vscode-ibmcloud-schematics/blob/main/extension-client/src/util/UserInput.ts#L58)

Once the user clicks `ok` we should open the above URL in browser using  `openExternal` , Reference in this [API doc](https://code.visualstudio.com/api/references/vscode-api)

ok or cancel we then show above message in terminal and be done